yeat/workflows/workflows.py

```diff
@@ -62,17 +61,22 @@ def check_canu_required_params(extra_args, cores):
         )
 
 
-def run_pacbio(fastq, assembly_configs, outdir=".", cores=1, sample="sample", dryrun="dry"):
+def run_pacbio(assembly_samples, assembly_configs, outdir=".", cores=1, dryrun="dry"):
     snakefile = resource_filename("yeat", "workflows/snakefiles/Pacbio")
-    fastq = Path(fastq).resolve()
-    if not fastq.is_file():
-        raise FileNotFoundError(f"No such file: '{fastq}'")
-    assemblers = [config.algorithm for config in assembly_configs]
-    extra_args = {config.algorithm: config.extra_args for config in assembly_configs}
+    samples = {k: v.sample for k, v in assembly_samples.items()}
+    labels = [config.label for config in assembly_configs]
+    assemblers = {config.label: config.algorithm for config in assembly_configs}
+    extra_args = {config.label: config.extra_args for config in assembly_configs}
+    label_to_samples = {config.label: config.samples for config in assembly_configs}
     if "canu" in assemblers:
         check_canu_required_params(extra_args["canu"], cores)
```

Comment:
Seems like this is validation. Probably belongs with other config validation code?

Reply:
Moved this in the `Assembler` class object. This check happens as soon as the `Assembler` class is instantiated.